tools/c7n_gcp/tests/data/flights/bucket-iam-policy/get-storage-v1-b-cloud-custodian-test-1-iam_1.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Guessing a lot of these test files are from the other PR. Lets try to seperate them if we can